localization/tryCam.py

```diff
@@ -0,0 +1,144 @@
+"""Cam funcs for setting configs and running Intel D455.
+
+Many appreciates to Ed for his config and streaming methods using Python in:
+https://github.com/autorope/donkeycar/blob/5e234c3101cc5f54935c240819e3840596c753a3/donkeycar/parts/realsense435i.py#L122
+Functions are mainly built on the original code written by him. I have modified some parameters 
```

Comment:
Elegant comment, love that. Also, can we integrate them into our code in ./hardware?

Reply:
I have moved cam_fetch.py and cam_funcs.py to ./hardware folder. gnss_fetch.py was not changed since the original file in ./hardware  would still be available if we set sentence from GGA to RMC manually.